JS/index.js

```diff
@@ -382,12 +380,13 @@ $(
                         $("#myCarousela ol").append("<li data-target='#myCarousela' data-slide-to='"+index+"'></li>");
                     }
 
+                    $("#myCarousela").carousel('cycle');
+
                     /*对应显示*/
                     $("#sideNavUl").find("li").mouseover(function () {
                         $(this).addClass("active").siblings().removeClass("active");
                         var index = $(this).index();
-                        $("#lunboBox>div").eq(index).show().siblings().hide();
-
+                        $("#myCarousela").carousel(index)
```

Comment:
instead of hiding other images, use the method provided by Bootstrap carousel to show the corresponding image